### DISCUSSION TITLE
iOS Failed to set code signing settings for ios/._Runner.xcodeproj

### DISCUSSION THREAD

**Discussion by ferdiangunawan** [codemagic-ci-cd/codemagic-docs#1686] [Q&A] [answered]
I got this error when xcode uses profile. I have checked xcode setting, everything seems ok. no red line error in sign-in and capabilities. Any help please?

<img width="741" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/49526345/2a2a7ed7-3dbb-4891-adfd-a53131b76313">

**Comment by icarusdust** [comment, marked as answer]
Hi @ferdiangunawan, from the screenshot, I can say that there is a typo in your project's xcodeproj file name. There should not be `._ `chars in front of it. Please remove them and the build should pass that step successfully. Let me know in case of any further questions.

**Reply by ferdiangunawan** [reply]
yes it was the problem. thank you!
